pkg/event/dispatcher.go

```diff
@@ -77,7 +72,11 @@ type QueueEventDispatcher struct {
 	eventFlushLock sync.Mutex
 	Dispatcher     Dispatcher
 
-	metrics Metrics
+	// metrics
+	queueSize         metrics.Gauge
+	sucessFlush       metrics.Counter
+	failFlushCounter  metrics.Counter
+	retryFlushCounter metrics.Counter
```

Comment:
Two of these fields have `Counter` in the field name, but one doesn't - seems weird.